.travis.yml

```diff
@@ -4,6 +4,8 @@ node_js:
   - "4"
   - "5"
   - "6"
+  - "7"
+  - "8" 
```

Comment:
Please could this be changed to the below to align with the [CLI](https://github.com/adaptlearning/adapt-cli/blob/develop/.travis.yml#L3-L4)?
```yml
node_js:
  - "lts/*"
```

Reply:
I think we should explicitly define what versions we test otherwise when a lts version jumps all our tests may stop working.

Reply:
I'd still be on board with having the dynamic LTS value.

I could see it as an advantage because by the time a version enters active LTS, it is pulled down by default for new Node downloads. The tests may help in flagging possible issues as they arise.

Happy to get a general consensus. Any thoughts @taylortom / @dancgray / @lc-thomasberger ?

Reply:
👍

Reply:
It seems that just setting the value as `lts/*` will cause travis to use the latest active LTS, which at this point in time is v8 -- which I don't have a problem with (once we fix our v8 issues), because it'll force us to adopt current releases quicker... 

There's not much documentation on this, and can't find anything that matches our specific needs.